src/aws-cpp-sdk-core/include/aws/core/utils/Array.h

```diff
@@ -77,29 +96,40 @@ namespace Aws
                 {
                     if(arr->m_size > 0 && arr->m_data)
                     {
-                        size_t arraySize = arr->m_size;
+                        size_t arraySize = arr->m_length;
+#ifdef _WIN32
+                        std::copy(arr->m_data.get(), arr->m_data.get() + arraySize, stdext::checked_array_iterator< T * >(m_data.get() + location, m_length));
```

Comment:
win throws error if + is unchecked? is there a way to write code that works for all platforms without ifdefs?

Reply:
I guess this is a side-effect of the rebase, this stdext usage has been recently removed from the SDK: https://github.com/aws/aws-sdk-cpp/pull/2726

Reply:
yeah removing again